core/soc/estimator.go

```diff
@@ -50,6 +50,18 @@ func (s *Estimator) Reset() {
 	s.energyPerSocStep = s.virtualCapacity / 100
 }
 
+// Estimate charge duration without using vehicle api
+func (s *Estimator) AssumedChargeDuration(targetSoC int, chargePower float64) time.Duration {
```

Comment:
update comment:

    // AssumedChargeDuration estimates charge duration up to targetSoC based on virtual capacity

Reply:
OK